tools/topology/topology1/development/sof-imx8-compr-pcm-wm8960.m4

```diff
@@ -0,0 +1,92 @@
+#
+# Topology with codec_adapter processing component for i.MX8QM/i.MX8QXP
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`sai.m4')
+include(`pcm.m4')
+include(`buffer.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include DSP configuration
+include(`platform/imx/imx8.m4')
+
+
+# Post process setup config
+
+ #codec Post Process setup config
+#
+# Define the pipelines
+#
+# PCM0 ----> volume -----> SAI3 (wm8960)
+#
+
+DECLARE_SOF_RT_UUID("passthrough_codec", passthrough_uuid, 0x376b5e44, 0x9c82,
+		0x4ec2, 0xbc, 0x83, 0x10, 0xea, 0x10, 0x1a, 0xf8, 0x8f);
+
+define(`CA_UUID', passthrough_uuid)
+
+# Post process setup config for playback
+define(`CA_SETUP_CONTROLBYTES',
+``	bytes "0x53,0x4f,0x46,0x00,0x00,0x00,0x00,0x00,'
+`	0x0C,0x00,0x00,0x00,0x00,0x10,0x00,0x03,'
+`	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
+`	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
+`	0x00,0x00,0x00,0x00,0x0C,0x00,0x00,0x00,'
+`	0x20,0x00,0x00,0x00"''
+)
```

Comment:
we don't need any parameters to be setup initially here. 

Similar with this:  https://github.com/thesofproject/sof/pull/6129/commits/c704769503b72fc7b5015cc73f694ce24cc466fb

So, we only need to add the header which is 32 bytes. And that's all for now.

Reply:
Thanks. Fixed.